providers/flagd/pkg/service/grpc/service.go

```diff
@@ -0,0 +1,175 @@
+package grpc_service
+
+import (
+	ctx "context"
+	"errors"
+
+	models "github.com/open-feature/flagd/pkg/model"
+	models2 "github.com/open-feature/golang-sdk-contrib/providers/flagd/pkg/model"
```

Comment:
Can we think of any better naming than `models`/`models2`?

Reply:
renamed to (package)Models